core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -222,8 +222,10 @@ class ContainerPool(
     final def getContainer(tryCount: Int, position: Long, key: ActionContainerId, conMaker: () => WhiskContainer)(implicit transid: TransactionId): ContainerResult = {
         val positionInLine = position - completedPosition.cur // Indicates queue position.  1 means front of the line
         val available = slack()
-        if (tryCount % 100 == 0) {
-            warn(this, s"""getImpl possibly stuck because still in line:
+        val warnAtCount = 200
+        val waitDur = 50.millis
+        if (tryCount == warnAtCount) {
```

Comment:
Maybe not but a constant stream of messages is not useful either.  If the queue is long, you are supposed to wait a while so that's not necessarily an error.